pom.xml

```diff
@@ -241,5 +242,18 @@
             <version>5.2.9.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+            <scope>provided</scope>
+        </dependency>
+        <!-- sun.misc.Unsafe package support required for java 1.8 compiler -->
+        <dependency>
```

Comment:
Why wasn't this necessary before? IIU https://github.com/headius/unsafe-mock correctly this is only necessary if you compile with a JDK older than JDK8 EA 17b which we don't support. Please get rid of this dependency. We should always use the original java.misc.Unsafe instead of this really old (8 year) fork!